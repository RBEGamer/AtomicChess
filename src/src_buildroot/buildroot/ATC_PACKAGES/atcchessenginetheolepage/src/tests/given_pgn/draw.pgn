[Event "?"]
[Site "Warsaw ol (08)"]
[Date "1935.??.??"]
[Round "?"]
[White "Alekhine, Alexander A"]
[Black "Steiner, Lajos J"]
[Result "1/2-1/2"]
[ECO "E12j"]

1. Ng1-f3 Ng8-f6 2. c2-c4 b7-b6 3. Nb1-c3 Bc8-b7 4. d2-d4 e7-e6 5. a2-a3
Bf8-e7 6. Bc1-f4 O-O 7. Qd1-c2 Nf6-h5 8. Bf4-d2 f7-f5 9. e2-e3 d7-d6 10.
Bf1-d3 g7-g6 11. e3-e4 Nb8-d7 12. O-O-O Nd7-f6 13. e4xf5 e6xf5 14. d4-d5
Nf6-d7 15. Bd2-h6 Nh5-g7 16. h2-h4 Be7-f6 17. Nf3-g5 Nd7-c5 18. Bd3-e2
Rf8-e8 19. Be2-f3 a7-a5 20. Rd1-e1 a5-a4 21. g2-g4 Nc5-b3+ 22. Kc1-b1
Re8xe1+ 23. Rh1xe1 Nb3-d4 24. Qc2-d3 Nd4xf3 25. Qd3xf3 f5xg4 26. Qf3xg4
Ng7-f5 27. Ng5-e6 Nf5xh6 28. Qg4-f4 Qd8-e7 29. Qf4xh6 Bf6xc3 30. b2xc3
Bb7-c8 31. h4-h5 Bc8xe6 32. Re1xe6 Qe7-h4 33. Re6xg6+ h7xg6 34. Qh6xg6+
Kg8-h8 35. Qg6-h6+ Kh8-g8 36. Qh6-g6+ Kg8-h8 37. Qg6-h6+ Kh8-g8 38. Qh6-g6+
1/2-1/2

