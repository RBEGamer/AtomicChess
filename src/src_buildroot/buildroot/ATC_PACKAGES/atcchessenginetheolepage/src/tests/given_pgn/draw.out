[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 3
[move]	board::PieceType: 0
[kingside castling]	color: 1
[move]	board::PieceType: 1
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 0
[queenside castling]	color: 0
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 6 rank: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 6 rank: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 2
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 2
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[check]	: color: 0
[move]	board::PieceType: 0
[move]	board::PieceType: 2
[piece_taken]	 piece: 2 file: 5 rank: 1
[check]	: color: 0
[move]	board::PieceType: 2
[piece_taken]	 piece: 2 file: 5 rank: 1
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 4
[piece_taken]	 piece: 3 file: 6 rank: 3
[move]	board::PieceType: 1
[piece_taken]	 piece: 4 file: 6 rank: 3
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 7 rank: 4
[move]	board::PieceType: 1
[piece_taken]	 piece: 5 file: 7 rank: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[piece_taken]	 piece: 3 file: 8 rank: 6
[move]	board::PieceType: 1
[move]	board::PieceType: 1
[move]	board::PieceType: 1
[piece_taken]	 piece: 4 file: 8 rank: 6
[move]	board::PieceType: 3
[piece_taken]	 piece: 4 file: 3 rank: 3
[move]	board::PieceType: 5
[piece_taken]	 piece: 3 file: 3 rank: 3
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[piece_taken]	 piece: 4 file: 5 rank: 6
[move]	board::PieceType: 2
[piece_taken]	 piece: 3 file: 5 rank: 6
[move]	board::PieceType: 1
[move]	board::PieceType: 2
[piece_taken]	 piece: 5 file: 7 rank: 6
[check]	: color: 1
[move]	board::PieceType: 5
[piece_taken]	 piece: 2 file: 7 rank: 6
[move]	board::PieceType: 1
[piece_taken]	 piece: 5 file: 7 rank: 6
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 1
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 1
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 1
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 1
[check]	: color: 1
[draw]	
[game_finished]	
